bot/src/com/mygdx/game/bot/screen/GameScreen.java

```diff
@@ -132,6 +136,7 @@ private void runBotIteration() {
     var field = nextFieldUtil.selectFieldInRangeOfUnit(unit);
     log.info("moving entity");
     moveEntityService.moveEntity(unit, coordinatesComponentMapper.get(field));
+    botAttackUtil.attack(unit);
   }
```

Comment:
czyli teraz jest tak:
scenariusz 1:
 - unit robi krok
 - akurat stanął blisko przeciwnika
 - atakuje

scenariusz 2:
 - przeciwnik wchodzi w zasięg
 - unit musi się ruszyć (potencjalnie przez to traci zasięg)
 - atakuje albo i nie, bo może stracił okazję

Czyli nie ma obsługi tego, żeby zaatakować przed zrobieniem ruchu. Ale można się w to nie bawić.

Reply:
No nie ma takiego czegoś, ale można założyć, że to najprostszy i najgłupszy bot, nie sprawdza nawet, czy przeżyje atak, a może byćtak, że unit ruszy się w to samo miejsce, i wtedy nie straci zasiegu

Reply:
ok